### ISSUE TITLE
NTR: CXCL10 elevated in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8432]
New term request CXCL10 elevated in blood (elthat_200619181935): CXCL-10; chemokine ligand  10; INP10; interferon-gamma-inducible protein 10

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000486
- label: Elevated circulating CXCL10 concentration
- parents: [Abnormality of chemokine secretion (HP:0011115)](https://hpo.jax.org/app/browse/term/HP:0011115)
- pmid: PMID:22038159
- def: The concentration of C-X-C motif chemokine 10 (CXCL10) in the blood circulation is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8432
- synonyms:
